src/CoreBundle/private/modules/CMSDocumentManager/CMSDocumentManager.class.php

```diff
@@ -290,22 +290,35 @@ public function removeConnection()
      */
     protected function GetDocumentRowHTML($documentID)
     {
-        $oFileDownload = new TCMSDownloadFile();
-        /** @var $oFileDownload TCMSDownloadFile */
-        $oFileDownload->Load($documentID);
+        $downloadDocument = new TCMSDownloadFile();
+        $downloadDocument->Load($documentID);
 
         $translator = $this->getTranslator();
 
+        $deleteButtonTemplate = '<button type="button" class="btn btn-danger btn-sm" onClick="if(confirm(\'%s\')){removeDocument(\'%s\',\'%s\',\'%s\',\'%s\')}">
+                                    <i class="far fa-trash-alt mr-2"></i>%s
+                                </button>';
+        $deleteButton = sprintf($deleteButtonTemplate,
+            TGlobal::OutJS($translator->trans('chameleon_system_core.field_download.confirm_removal', [], \ChameleonSystem\CoreBundle\i18n\TranslationConstants::DOMAIN_BACKEND)),
```

Comment:
Please import the TranslationConstants class.

Reply:
done